hydra-node/exe/tx-cost/TxCost.hs

```diff
@@ -1,19 +1,20 @@
-{-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE TypeApplications #-}
 
 module TxCost where
 
 import Hydra.Prelude hiding (catch)
 
+import qualified Cardano.Api.UTxO as UTxO
 import Cardano.Binary (serialize)
 import qualified Cardano.Ledger.Alonzo.PParams as Ledger
 import qualified Cardano.Ledger.Alonzo.Scripts as Ledger
 import qualified Data.ByteString.Lazy as LBS
-import Data.Fixed (E2, Fixed)
+import Data.Fixed
```

Comment:
Just curious, why not explicit imports here?

Reply:
Seems like an oversight, sorry.